<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsultarDisponibilidadeDialog</class>
 <widget class="QDialog" name="ConsultarDisponibilidadeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(234, 240, 242);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/imagens/logoo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="LocalidadecomboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="tipoQuartocomboBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="checkInDateEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>170</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="numDiariasSpinBox">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>170</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Onde</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Quarto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>150</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Check_In</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>150</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Di√°rias</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 17pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Jericoacoara</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 17pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Canoa_Quebrada</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>260</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 17pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Cumbuco</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/imagens/1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/imagens/3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/imagens/2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>570</y>
     <width>141</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/imagens/5.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>570</y>
     <width>181</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/imagens/6.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>570</y>
     <width>131</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/imagens/4.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>290</y>
     <width>121</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/imagens/9.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>290</y>
     <width>151</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/imagens/8.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>290</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/imagens/7.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>520</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Feedback</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>548</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>‚≠ê‚≠ê‚≠ê‚≠ê‚≠ê</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>572</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Tamara A:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>572</y>
     <width>421</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>O lugar faz jus ao nome. Acordar com o som dos passarinhos e tomar caf√© </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>590</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>com vista para o mar foi simplesmente m√°gico!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>632</y>
     <width>421</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Fiquei impressionado com o cuidado nos detalhes. Desde a decora√ß√£o</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>650</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>r√∫stica at√© o cheirinho do quarto... tudo perfeito!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>632</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Igor Tavares:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>610</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>‚≠ê‚≠ê‚≠ê‚≠ê‚≠ê</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>702</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Jos√© Edu.:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>702</y>
     <width>421</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>√â como se voc√™ entrasse em outro mundo. A conex√£o com a natureza me </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>680</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>‚≠ê‚≠ê‚≠ê‚≠ê‚≠ê</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>720</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>renovou. A lua refletindo no lago √† noite √© inesquec√≠vel.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>762</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Gabrielle k:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>762</y>
     <width>421</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Experi√™ncia incr√≠vel! Fiz trilhas, tirei fotos com os animais e ainda consegui </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>780</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>relaxar numa rede debaixo do c√©u estrelado.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>740</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>‚≠ê‚≠ê‚≠ê‚≠ê‚≠ê</string>
   </property>
  </widget>
  <widget class="QPushButton" name="consultarButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>170</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>üîç</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>______________________________________________________________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>______________________________________________________________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>801</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>______________________________________________________________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>230</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>______________________________________________________________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>490</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>______________________________________________________________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>800</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;Liberation Serif&quot;;</string>
   </property>
   <property name="text">
    <string>______________________________________________________________________</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
